docs/docusaurus/src/components/WasThisHelpful/index.js

```diff
@@ -51,20 +53,29 @@ export default function WasThisHelpful(){
             $survey_id: '018dd725-c595-0000-00c6-6eec1b197fd0'
         })
         setIsOpen(false)
+        setError(false)
     }
 
-    const sendReview = (e) => {
+    const sendReview = async (e) => {
         e.preventDefault()
-        if(formData.description){
-            posthog.capture("survey sent", {
-                $survey_id: '018dd725-c595-0000-00c6-6eec1b197fd0',
-                $survey_response: formData.name,
-                $survey_response_1: formData.email,
-                $survey_response_2: formData.description,
-                $survey_response_3: pathname,
-                $survey_response_4: formData.selectedValue.replaceAll('-', ' ')
-            })
-            setIsOpen(false)
+        if (formData.description) {
```

Comment:
Should we set error to false before we hit the API if it was previously marked as `true`?